main/glossary/README.md

```diff
@@ -424,9 +421,14 @@ can immediately cause the [seat](#seat) to exit, getting back the amount it offe
 
 ## Passable
 
-A *passable* is something that can be marshalled (see the
-[Marshaling section in the JavaScript Distributed Programming Guide](/guides/js-programming/far.md#marshaling-by-copy-or-by-presence))
-and sent to and from remote objects.
+A *passable* is something that can be sent to and from remote objects.
+Passables include pass-by-copy primitive values such as numbers and strings and
+pass-by-reference values such as Remotables and Promises.
+Passables also include hardened acyclic pass-by-copy containers that recursively terminate
+in non-container passables, such as CopyArrays like `harden(['foo', 'bar'])` and
+CopyRecords like `harden({ keys: [0, 1], values: ['foo', 'bar'] })`.
```

Comment:
I found this framing a little opaque so I’m suggesting a break-down:
```suggestion
Values that are pass-by-copy include:
* Primitive values including `string`, `number`, `bigint`, `boolean`, `null`, and `undefined`,
  * Agoric _tentatively_ supports registered symbols, as in `Symbol.for(name)`, though we advise against using them,
  * Agoric _even more tentatively_ supports well-known-symbols like `Symbol.asyncIterator`, though we strongly advise against using them,
* `CopyArray`: [hardened](#harden, acyclic, arrays like `harden(['foo', 'bar'])`,
* `CopyRecord`: hardened, acyclic objects like `harden({ keys: [0, 1], values: ['foo', 'bar'] })`,
* `CopyTagged`: [tagged](#tagged) pass-by-copy values like `{ [Symbol.for('passStyle')]: 'tagged', tag: 'copySet', payload: [1, 2, 3] }`.
```

I have not verified `#tagged` exists.

Reply:
Again, please, no novel info in the glossary. If we're going to have such details, let's have them in the [section on passables](https://docs.agoric.com/guides/js-programming/far.html#pass-styles-and-harden).